site/src/css/components/_LandingPage.scss

```diff
@@ -0,0 +1,95 @@
+/* LandingPage CSS */
+
+.landing-page {
```

Comment:
So this is a comment for all of the scss. In general we shouldn't really nest more than 3 levels because understanding exactly how the CSS is going to behave starts getting tricky and it could end up doing weird things if the html structure at all changes. Since we're relying on docusaurus and stuff I think this is even more relevant.

If we're going to nest like this you could add & to the next level like I have in the example below. This will create a new selector `landing-page-left` that you could reference above like I mentioned in an earlier comment. This way even if the html changes slightly that selector should still accurately hit the element it's supposed to. (Note that if we do this then you can't search the scss files for `landing-page-left` since it won't exist like that, but that shouldn't be a problem in our case)
```
    &-left {
```